vms/platformvm/add_validator_tx_test.go

```diff
@@ -27,6 +27,226 @@ import (
 	"github.com/chain4travel/caminogo/vms/secp256k1fx"
 )
 
+func TestAddValidatorTxWithNewReward(t *testing.T) {
+	vm, _, _ := defaultVM()
+	vm.ctx.Lock.Lock()
+	defer func() {
+		if err := vm.Shutdown(); err != nil {
+			t.Fatal(err)
+		}
+		vm.ctx.Lock.Unlock()
+	}()
+
+	key, err := vm.factory.NewPrivateKey()
+	if err != nil {
+		t.Fatal(err)
+	}
+	nodeID := key.PublicKey().Address()
```

Comment:
nodeID != address
Its ID uniq for machine where node runs
You can get it from console log when you starts the node

Reply:
Well, if you use it just to get some random ids.ShortID, than its probably fine, just plz clarify that with comment

Reply:
Oh, I see.. Ava guys do the same.. Well, I hope they knew the difference and just used it as some random ids.ShortID generator. Lets keep the same look then